CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+<!-- 8.5.15 START -->
+
+# 8.5.15 (2022-11-08)
+
+### Features and enhancements
+
+- **Chore:** Upgrade Go to 1.19.2. [#56857](https://github.com/grafana/grafana/pull/56857), [@sakjur](https://github.com/sakjur)
+- **Chore:** Upgrade Go to 1.19.3. [#58070](https://github.com/grafana/grafana/pull/58070), [@sakjur](https://github.com/sakjur)
```

Comment:
This is not part of 8.5.15, right?

Reply:
Ah you are right. Will fixup here and `main`